docs/public/installation/zonemaster-backend.md

```diff
@@ -711,7 +715,53 @@ Now go back to "[Database configuration](#53-database-configuration-freebsd)"
 to create the database tables and then continue with the steps after that.
 
 
-## 9. Cleaning up the database
+## 9. Installation with Clickhouse (experimental)
+
+> This is an experimental feature.
+
+First follow the installation instructions for the OS in question, and then go
+to this section to install Clickhouse.
+
+To install Clickhouse, follow the instructions from the [official
+documentation](https://clickhouse.com/docs/en/install#available-installation-options).
+
+Install the Perl bindings (Clickhouse relies on the MySQL DBI):
+
+* Rocky Linux: `sudo dnf install -y perl-DBD-MySQL`
+* Debian/Ubuntu: `sudo apt install libdbd-mysql-perl`
+* FreeBSD: `pkg install mysql57-server p5-DBD-mysql`

```

Comment:
Why is the installation of MySQL server included for FreeBSD when it is not for the other OSs? It should not be run, should it?

Reply:
It's an error. I've removed it.